### DISCUSSION TITLE
水月肉鸽

### DISCUSSION THREAD

**Discussion by timelaijnjadk** [MaaAssistantArknights/MaaAssistantArknights#4380] [General]
水月第一层 互助 （就是开局有一个大盾，然后有浮空生物）经常显示关卡识别错误！！

**Comment by MistEO** [comment]
更新一下

**Comment by Wabrielny** [comment]
我也遇到了相同的问题

**Comment by shendeguize** [comment]
水月关卡识别错误+1

**Comment by MistEO** [comment]
提个 issue 发个 log 看看

**Reply by Optimistz** [reply]
![屏幕截图 2023-04-28 203132.png](https://user-images.githubusercontent.com/97942234/235327731-6da16dca-3ada-487a-b6df-cdacee805138.png)![屏幕截图 2023-04-30 065615.png](https://user-images.githubusercontent.com/97942234/235327736-86c16307-9a7d-4701-bf56-d236eaf9da17.png)![屏幕截图 2023-04-30 065603.png](https://user-images.githubusercontent.com/97942234/235327734-a28ae4cd-8084-48f3-abc0-6c14faf1b419.png)
不太会提取，只能截图了。关卡为:![Screenshot_2023_0430_071754.jpg](https://user-images.githubusercontent.com/97942234/235327875-f8d53030-65df-41d3-8a63-d1142fa23a11.jpg)

**Comment by MistEO** [comment]
修复了，再更新一下（）
